### PR TITLE
Update exception log messages to include all available data

### PR DESCRIPTION
**Pull request by jceresini**
Fix exceptions that were classified as debug logs. Add more context to the exception messages by reusing the existing log object when reporting exceptions

### CODE REVIEW

File: app/run.py [hunk 1]
```diff
@@ -160,11 +160,11 @@ def callback(pubsub_message):
                 )
             resource = Resource.factory('gcp', asset_info, credentials=project_creds)
         except Exception as e:
-            logger.debug({
-                'log_id': log_id,
-                'message': 'Internal failure in rpe-lib',
-                'details': str(e)
-            })
+            import traceback
```
Comment (rvandegrift) [inline app/run.py]: can you move these imports to the usual spot in the top of the file?

File: app/run.py [hunk 2]
```diff
@@ -174,15 +174,16 @@ def callback(pubsub_message):
...
+            log['violation_count'] = len(violations)
             log['remediation_count'] = 0
+            log['violations'] = {str(v):{'remediated': False} for _,v in violations}
         except Exception as e:
-            logger.debug({
-                'log_id': log_id,
-                'message': 'Execption while checking for violations',
-                'details': str(e)
-            })
+            import traceback
```
Comment (rvandegrift) [inline app/run.py]: ditto

File: app/run.py [hunk 3]
```diff
@@ -222,14 +225,11 @@ def callback(pubsub_message):
...
                 # Otherwise they start to fill up the buffer of unacknowledged messages
-                logger({'log_id': log_id,
-                        'message': 'Exception while attempting remediation of {}'.format(violation),
-                        'details': str(e)})
-
-                if 'exceptions' not in log:
-                    log['exceptions'] = []
-
-                log['exceptions'].append(str(e))
+                import traceback
```
Comment (rvandegrift) [inline app/run.py]: ditto

File: app/run.py (hunk 2 above)
Comment (jceresini) [reply app/run.py]: done

**Pull request merged**